lib/scrabble/scrabble.rb

```diff
@@ -1,4 +1,4 @@
-#require "player.rb"
+require "./lib/scrabble_module"
 module Scrabble
   class Scrabble
```

Comment:
It looks like the "magic number" of 7 is used a lot, so it might be a very good place to use a constant